### PR TITLE
Feature/add is default flag to address

### PR DESCRIPTION
**Pull request by gengjozsef**

### CODE REVIEW

File: src/FondOfSpryker/Zed/JellyfishB2B/Business/JellyfishB2BBusinessFactory.php [hunk 1]
```diff
@@ -227,6 +229,17 @@ protected function createCompanyBusinessUnitAddressExpanderPlugin(): JellyfishCo
         );
     }
 
+    /**
+     * @return \FondOfSpryker\Zed\JellyfishB2B\Dependency\Plugin\JellyfishCompanyBusinessUnitExpanderPluginInterface
+     */
+    protected function createCompanyBusinessUnitDefaultAddressExpanderPlugin(): JellyfishCompanyBusinessUnitExpanderPluginInterface
+    {
+
```
Comment (julianzimmermann) [inline src/FondOfSpryker/Zed/JellyfishB2B/Business/JellyfishB2BBusinessFactory.php:237]: remove new line

**Review by julianzimmermann — APPROVED** [review]

File: src/FondOfSpryker/Zed/JellyfishB2B/Communication/Plugin/JellyfishCompanyBusinessUnitDefaultAddressExpanderPlugin.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+    /**
+     * @param \Generated\Shared\Transfer\JellyfishCompanyBusinessUnitTransfer $jellyfishCompanyBusinessUnitTransfer
+     *
+     * @return \Generated\Shared\Transfer\JellyfishCompanyBusinessUnitTransfer
+     */
+    public function expand(
+        JellyfishCompanyBusinessUnitTransfer $jellyfishCompanyBusinessUnitTransfer
+    ): JellyfishCompanyBusinessUnitTransfer {
+
```
Comment (daniel-rose) [inline src/FondOfSpryker/Zed/JellyfishB2B/Communication/Plugin/JellyfishCompanyBusinessUnitDefaultAddressExpanderPlugin.php:40]: remove new line

**Review by daniel-rose — CHANGES_REQUESTED** [review]

File: src/FondOfSpryker/Zed/JellyfishB2B/Communication/Plugin/JellyfishCompanyBusinessUnitDefaultAddressExpanderPlugin.php (hunk 2 above)
Comment (gengjozsef) [reply src/FondOfSpryker/Zed/JellyfishB2B/Communication/Plugin/JellyfishCompanyBusinessUnitDefaultAddressExpanderPlugin.php]: done

File: src/FondOfSpryker/Zed/JellyfishB2B/Business/JellyfishB2BBusinessFactory.php (hunk 1 above)
Comment (gengjozsef) [reply src/FondOfSpryker/Zed/JellyfishB2B/Business/JellyfishB2BBusinessFactory.php]: done

**Review by daniel-rose — APPROVED** [review]

**Pull request merged**